ResearchKit/Common/ORKAnswerFormat.h

```diff
@@ -151,8 +151,8 @@ ORK_CLASS_AVAILABLE
 
 + (ORKTextAnswerFormat *)textAnswerFormatWithMaximumLength:(NSInteger)maximumLength;
 
-+ (ORKTextAnswerFormat *)textAnswerFormatWithValidationRegex:(NSString *)validationRegex
-                                              invalidMessage:(NSString *)invalidMessage;
++ (ORKTextAnswerFormat *)textAnswerFormatWithValidationRegularExpression:(NSRegularExpression *)validationRegularExpression
```

Comment:
Deprecation flags serve a valuable purpose in any API. It allows other developers to pin to master and use the latest build (which often includes bug fixes) without the worry that it's going to break their own code, and it serves as a document of the change you are making and how the developer can modify their code to conform to the new method.